src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -18,7 +18,7 @@
 
 helps['acr repository'] = """
     type: group
-    short-summary: Manage repositories for Azure Container Registries.
+    short-summary: Manage repositories (image names) for Azure Container Registries.
```

Comment:
this seems wrong since you can do things like create list delete on the actual repositories, not related to images at all

Reply:
I'm just suggesting repositories are equivalent to image names...

Reply:
sorry, I thought this was on registry for some reason initially, just ignore this. I even repeated "repositories" :nauseated_face: